### ISSUE TITLE
Ingame wiki / faq / knowledgebase

### ISSUE THREAD

**Issue by Fohlen** [inexorgame/inexor-core#262]
_From @Fohlen on December 5, 2015 20:7_

We could render an in-game wiki on the website with markdown.
The files should go into `blog-data` and could also be available in the game using a NodeJS/markdown-renderer.
On our websites there should be a sort of knowledge-base, searchable with Ajax

_Copied from original issue: inexor-game/website#30_

**Comment by Fohlen** [comment]
_From @Croydon on December 5, 2015 22:13_

What should be the content of such a wiki?

**Comment by Fohlen** [comment]
Could be anything from general tips, tweaking settings, explanation on gamemodes or coop-edit guides..

**Comment by Fohlen** [comment]
_From @Croydon on December 8, 2015 20:1_

A wiki for these purposes are imo a post-stable job.

**Comment by MartinMuzatko** [comment]
What do you think about an in-game browser like Steam has? 
http://media.steampowered.com/apps/client/manage_screenshots_overlay.jpg

**Comment by Croydon** [comment]
Well, we are integrating CEF and a new UI on top of that. Therefore, we will have an ingame browser. To what extend we will use it, well, that is another story.

**Comment by Fohlen** [comment]
This is related to #382

**Comment by Fohlen** [comment]
Will this be a separate section of the wiki? @Croydon

**Comment by Croydon** [comment]
In my opinion we should only touch this issue when we starting to work on gameplay and other high level stuff. 

I also didn't suggest that @Fohlen. But I would advise strictly against using our main wiki for it. It is already big and will grow bigger. It should stay clearly technically to not mix things up.

@Fohlen You suggested in the issue to use the blog-data repository for it. That would be one way.

**Comment by Fohlen** [comment]
blog-data seems inconvenient, rather create a new "knowledge" repository than.

**Comment by Fohlen** [comment]
This will be closed via #

**Comment by MartinMuzatko** [comment]
via which issue? D: @Fohlen

**Comment by a-teammate** [comment]
yeah, I think #526 is related, but it may not be really focused on the aspect to integrate the docs ingame?

**Comment by aschaeffer** [comment]
As first step we could integrate the user documentation in `ui-flex`.
